platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -0,0 +1,485 @@
+    ##     API_KEY: The authentication is done using an API key added to the `X-Api-Key` header.
+    ##     JWT_TOKEN: The authentication is done using a JWT token added to the `Authorization`
+    ##         header. This token is generated from the secret key.
+    ##
+    ## If the type is not NONE, the secret must be set by the
+    ## environment variable: `secret.callback_api.auth_secret`
+    ##
+    ## The value of the secret depends on the type specified.
+    ##
+    ##    If type is JWT_TOKEN - the secret key of JWT encryption
+    ##    If type is API_KEY: - CallbackAPI requests (`Platform->Anchor`) will contain an
+    ##        authentication header whose token was built using this secret. The Anchor Backend will
+    ##        also store this same secret and use it to decode the incoming token to verify it
+    ##        came from the Platform.
+    #
+    type: NONE
+
+    ## @param: expiration_milliseconds
+    ## @type: int
+    ## Expiration time, in milliseconds, that will be used to build and validate the JWT tokens
+    #
+    expiration_milliseconds: 30000
+
+platform_api:
+  auth:
+    ## @param: type
+    ## @supported_values: NONE, API_KEY, JWT_TOKEN
+    ##     NONE: no authentication is used
+    ##     API_KEY: The authentication is done using an API key added to the `X-Api-Key` header.
+    ##     JWT_TOKEN: The authentication is done using a JWT token added to the `Authorization`
+    ##         header. This token is generated from the secret key.
+    ##
+    ## If the type is not NONE, the secret must be set by the
+    ## environment variable: `secret.callback_api.auth_secret`
+    ##
+    ## The value of the secret depends on the type specified.
+    ##
+    ##    If type is API_KEY: -
+    ##        PlatformAPI requests (`Anchor->Platform`) will contain an
+    ##        authentication header whose token was built using this secret. The Platform Server will
+    ##        use this secret to decode the incoming token to verify it came from the Anchor.
+    ##        also store this same secret and use it to decode the incoming token to verify it
+    ##        came from the Platform.
+    #
+    type: NONE
+
+
+payment_observer:
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable a circle payment observer
+  #
+  enabled: false
+
+  ## @param: circle_url
+  ## @type: string
+  ## Circle api endpoint.
+  #
+  circle_url: https://api-sandbox.circle.com
+
+
+  ## @param: tracked_wallet
+  ## @type: string
+  ## The id of the wallet to track. 'all' indicates that events for all wallets are tracked.
+  #
+  tracked_wallet: all
+
+
+## @param: languages
+## @supported_values: en
+## The supported languages defined by RFC4646 (https://datatracker.ietf.org/doc/html/rfc4646)
+#
+languages: en
+
+logging:
+  ## @param: level
+  ## @type: string
+  ## The root logger logging level
+  level: INFO
+
+  ## @param: level
+  ## @type: string
+  ## The "org.stellar" logger logging level.
+  stellar_level: INFO
```

Comment:
Whats the difference between these two loggers?

Reply:
The `logging.level` affects all logging messages while `logging.stellar_level` only affects messages printed by the classes inside `org.stellar.*` package.